heroic-component/src/test/java/com/spotify/heroic/tracing/RandomParentSamplerTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.spotify.heroic.tracing
+
+import io.opencensus.trace.*
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertFailsWith
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class RandomParentSamplerTest {
+
+    lateinit var sampler: Sampler
+    @Before
+    fun setup() {
+         sampler = RandomParentSampler(1.0)
```

Comment:
For useful testing I think you would have to pass in the random generator as a sampler argument, letting you mock it here. I'm undecided about whether that's worth doing.